builtin/logical/pki/path_ocsp.go

```diff
@@ -68,16 +68,33 @@ var (
 )
 
 func buildPathOcspGet(b *backend) *framework.Path {
-	return buildOcspGetWithPath(b, "ocsp/"+framework.MatchAllRegex(ocspReqParam))
+	pattern := "ocsp/" + framework.MatchAllRegex(ocspReqParam)
+
+	displayAttrs := &framework.DisplayAttributes{
+		OperationPrefix: operationPrefixPKI,
+		OperationVerb:   "query-with",
```

Comment:
Maybe drop the `with` here, and add it as `ocsp-with-get-req`? 

We've got some issues with OCSP GET requests though, that Steve's working on... I'd like to try to discourage that one if possible, and have people use the POST variant instead.

Reply:
We could mark it as Deprecated: true in the openapi to not show up in the generated libraries :)

Reply:
Adjusted to new names, thanks for suggestion!